docs/_layouts/about.html

```diff
@@ -1,46 +1,51 @@
+{% assign navpage = page.url | slice: 6, 3 %}
 <!doctype html>
-<html>
-
+<html lang="en">
 {% include head.html %}
 
 <body>
-
-{% include top.html %}
-
-<div id="documentation-container">
-
-<div id="sidebar-left-container" class="flex1 flex-right">
-<div id="sidebar-left">
-<h1 class="hidden">.</h1>
-{% include link name="Introduction" path="/about/" %}
```

Comment:
Seems like all the links have been inlined. In that case, docs/_includes/link can be removed

Reply:
I think I was overreaching too much with the inline links, the next PR goes back to the templated links, therefore, docs/_includes/link will need to stay.